README.md

```diff
@@ -179,26 +173,44 @@ Grype lets you define custom output formats, using [Go templates](https://golang
 
 - Specify the path to the template file (`-t ./path/to/custom.template`).
 
-- Grype's template processing uses the same data models as the `json` output format — so if you're wondering what data is available as you author a template, you can use the output from `grype <image> -o json` as a reference.
 
-**Example:** You could make Grype output data in CSV format by writing a Go template that renders CSV data and then running `grype <image> -o template -t ~/path/to/csv.tmpl`.
+**Example:** Suppose you want Grype to output specific data columns, in CSV format.
+
+(1) Grype's input template processing uses the same data models as found in the `json` output format Grype produces. So if you're wondering what data is available for you to use in a template, examine data objects found in the output file.
+
+```grype -o json SAMPLE.war  > SAMPLE.war.json```
+
+Refer to the section above [Basic Grype Vulnerability Data Shape](#basic-grype-vulnerability-data-shape) while looking inside the JSON objects found in `SAMPLE.war.json`. You will see vulnerability objects containing nested information. To use these any of these JSON values in the Golang template, you will notice a naming correspondence.
+* For example, the template expression `{{.Vulnerability.Fix.Versions}}` relates to the JSON object, ` "vulnerability": { "fix": { "versions": [ ] ` .
+* The template expression `{{- range .Matches}}` says to iterate across the JSON array, ` "matches":` .
+* More completely, below is what a custom `csv.tmpl` template file might contain:
 
-Here's what the `csv.tmpl` file might look like:
 ```gotemplate
-"Package","Version Installed","Vulnerability ID","Severity"
+"Artifact","Severity","Is Fixed","Vulnerability","Version","Version Fixed"
 {{- range .Matches}}
-"{{.Artifact.Name}}","{{.Artifact.Version}}","{{.Vulnerability.ID}}","{{.Vulnerability.Severity}}"
+"{{.Artifact.Name}}","{{.Vulnerability.Severity}}","{{.Vulnerability.Fix.State}}","{{.Vulnerability.ID}}","'{{.Artifact.Version}}","{{.Vulnerability.Fix.Versions}}"
```

Comment:
It's not clear to me why these column names need tweaking.

I like the addition of fix information, though. 👍

Reply:
"Tweaking" : I made a mistake. At your suggestion, I reduced the number of cols printed ... but accidentally left in spurious column headers. Thus, the "tweak" comment.